src/ray/object_manager/plasma/store.cc

```diff
@@ -158,6 +160,10 @@ PlasmaError PlasmaStore::HandleCreateObjectRequest(const std::shared_ptr<Client>
     return PlasmaError::OutOfMemory;
   }
 
+  if (fs_monitor_.OverCapacity()) {
+    RAY_LOG(ERROR) << "device_num != 0 but CUDA not enabled";
+    return PlasmaError::OutOfMemory;
```

Comment:
Could we return a new error for OutOfDisk? Then we can have a nicer/more specific error message directing users on how to fix it.

Reply:
Also, I think we should only do this if we're out of memory too, right? Since we may actually be able to create the object, even if we're over disk capacity.

Reply:
Yup. I'm debating if we treat out of disk/out of memory as same error or different ones... today it's treaded as out of memory. maybe it's time to create a new one!